### ISSUE TITLE
a11y: missing contrast for admin items in dark mode

### ISSUE THREAD

**Issue by dctalbot** [target/goalert#2200]
**Describe the Bug:**
Missing contrast.

**Steps to Reproduce:**
1. Go to /admin/config in dark mode and try to change a setting.

**Expected Behavior:**
Potentially remove the yellow highlight feature entirely, otherwise adjust colors as needed

**Observed Behavior:**
<img width="1015" alt="Screen Shot 2022-02-25 at 9 55 57 AM" src="https://user-images.githubusercontent.com/17692467/155746771-209500db-4059-47af-90b8-20a169bd81b3.png">

**Comment by Forfold** [comment]
Looks like the diff when saving, highlighting text in red and green, will also need to be adjusted
